src/main/java/com/jagrosh/jmusicbot/queue/FairQueue.java

```diff
@@ -20,6 +20,8 @@
 import java.util.List;
 import java.util.Set;
 
+import net.dv8tion.jda.api.entities.Message;
```

Comment:
FairQueue is a very generic container and should definitely not contain any references to JDA, JMusicBot, or any music-related systems at all. Do not modify this class unless it is to add generic functionality.